viewing_party/party.py

```diff
@@ -0,0 +1,156 @@
+
+#Test Wave 01 functions
+
+#Function returns a dictionary containing movie details
+def create_movie(movie_title, genre, rating):
+    if movie_title and genre and rating:
+        movie={
+            "title": movie_title,
+            "genre": genre,
+            "rating" : rating
+        }
+        return movie
+    else:
+        return None
+
+#Function to add movie to user's watched list    
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+#Function to add movie to user's watchlist
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+#Function to move movie from user's watchlist to watched list
+def watch_movie(user_data, movie_title):
+    for movie in user_data["watchlist"]:
+        for key, movie_tit in movie.items():
+            if movie_tit == movie_title:
+                user_data["watched"].append(movie)
+                user_data["watchlist"].remove(movie)       
```

Comment:
We don't need to loop through a dictionary if we already know the key you want. 
```suggestion
        if movie["title"] == movie_title:
            user_data["watched"].append(movie)
            user_data["watchlist"].remove(movie)       
```